django/test/testcases.py

```diff
@@ -828,6 +828,15 @@ class TransactionTestCase(SimpleTestCase):
     # This can be slow; this flag allows enabling on a per-case basis.
     serialized_rollback = False
 
+    # This attribute is strongly linked to serialized_rollback parameter.
```

Comment:
Wrap comments at 80 characters.

Reply:
Not sure to get your comment about 80 character limit. None of the comment lines exceed this limit. Do I miss something?

Reply:
Sorry, I wasn't clear - in this case I think that some of the comment block was wrapped at less than 80 characters, but too aggressively, e.g. If wrapped at 72 characters but the next line has a four letter word...

Reply:
Got it! Will do!